flexget/ui/src/plugins/series/series.component.js

```diff
@@ -1,90 +1,120 @@
+    'use strict';
 
-  angular
+    angular
     .module('flexget.plugins.series')
     .component('seriesView', {
-      templateUrl: 'plugins/series/series.tmpl.html',
-      controllerAs: 'vm',
-      controller: seriesController,
+        templateUrl: 'plugins/series/series.tmpl.html',
+        controllerAs: 'vm',
+        controller: seriesController,
     });
 
-  function seriesController($http, $mdDialog, seriesService) {
-    var vm = this;
+    function seriesController($http, $mdDialog, seriesService, $timeout) {
+        var vm = this;
+
+        var options = {
+            page: 1,
+            page_size: 10,
+            in_config: 'all',
+            sort_by: 'show_name'
+        }
+
+        vm.searchTerm = "";
+
+        function getSeriesList() {
+            seriesService.getShows(options).then(function(data) {
+                vm.series = data.shows;
+
+                vm.currentPage = data.page;
+                vm.totalShows = data.total_number_of_shows;
+                vm.pageSize = data.page_size;
+            });
+        }
+
+        vm.forgetShow = function(show) {
+            seriesService.deleteShow(show).then(function(data) {
+                getSeriesList();
+            });
+        }
+
+        /*vm.forgetShow = function(show) {
+        //Construct the confirmation dialog
+        var confirm = $mdDialog.confirm()
+        .title('Confirm forgetting show.')
+        .htmlContent("Are you sure you want to completely forget <b>" + show.show_name + "</b>?<br /> This will also forget all downloaded releases.")
+        .ok("Forget")
+        .cancel("No");
+
+        //Actually show the confirmation dialog and place a call to DELETE when confirmed
+        $mdDialog.show(confirm).then(function() {
+        $http.delete('/api/series/' + show.show_id, { params: { forget: true } })
+        .success(function(data) {
+        var index = vm.series.indexOf(show);
+        vm.series.splice(index, 1);
+    })
+    .error(function(error) {
+
+    //Show a dialog when something went wrong, this will change in the future to more generic error handling
+    var errorDialog = $mdDialog.alert()
+    .title("Something went wrong")
+    .htmlContent("Oops, something went wrong when trying to forget <b>" + show.show_name + "</b>:\n" + error.message)
+    .ok("Ok");
+
+    $mdDialog.show(errorDialog);
+})
+});
+}*/
+
+//Call from the pagination to update the page to the selected page
+vm.updateListPage = function(index) {
+    options.page = index;
+
+    getSeriesList();
+}
 
-    var options = {
-      page: 1,
-      page_size: 10,
-      in_config: 'all',
-      sort_by: 'show_name'
+vm.search = function() {
+    if(vm.searchTerm) {
+        seriesService.seriesShows(vm.searchTerm).then(function(data) {
+            vm.series = data.shows;
+        });
+    } else {
+        options.page = 1;
+        getSeriesList();
     }
+}
 
-    vm.searchTerm = "";
+vm.showEpisodes = function (show) {
+    vm.selectedShow = null;
```

Comment:
I'm not sure if you planned it already, but a check to see if this show is already selected, and preventing the list from reopening (and reloading data) could be nice.